communications/newsletters/newsletter_26_June2021.md

```diff
@@ -0,0 +1,175 @@
+The Turing Way Newsletter: 30 June 2021
+
+# A catchy title that sums the main points
+
+Hello Turing Way friends!
+
+In June, amidst the pandemic and lockdowns, our incredible Book Dash contributors and reviewers continued to collaborate and work on their contributions over the last month. 💪
+
+Thanks to them (you! 🌷):
+- 🎊 we published 8 chapters (with 20 subchapters) across guides for communication, collaboration and ethical research! 
+- 📹 we also published videos from the Community Share out events that include stories and ideas behind the illustrations we recently released on [Zenodo](https://zenodo.org/record/3332807).
+
+If you are wondering where to start, drop [Malvika an email](mailto:msharan@turing.ac.uk) who would love to chat about your work and find the right role for you in _The Turing Way_.
+
+Find more details on these topics (and more) below 👇
+
+## Community Meetings
+
+![A group with three people sitting across the table and working together while discussing something](images/2021-06-meeting.png)
+
+*An illustration of a group coworking. [Royalty-free image from Many Pixels](https://www.manypixels.co/gallery/)*
+
+You are invited to attend one or multiple of these online calls and community meetings: 
+
+* **Onboarding and Catch-up** calls are hosted to have open discussions with new and existing community members and onboard different people in _The Turing Way_ project.
+These calls will take place from 15:00 to 16:00 London time ([see in your local time](http://time.unitarium.com/utc/14)) on 2, 9, 16, 23 and 30 July.
+
+* **Collaboration Cafes** will be hosted on 7 and 21 July from 15:00 - 17:00 London time ([see in your local time](http://time.unitarium.com/utc/14)). 
+  - These are 2-hour long calls that feature discussions on topics of shared interest in _The Turing Way_ and several short quiet working sessions (see [the chapter](https://the-turing-way.netlify.app/community-handbook/coworking.html)).
+
+* **Weekly Coworking** calls are 1-hour long calls where you can catch up with _The Turing Way_ team members and other contributors and get some of your tasks done that need focused time or feedback.
+  - Join us from 11:00 - 12:00 ([see in your local time](http://time.unitarium.com/utc/10)) London time on 5, 12, 19 and 26 July 2021 (compatible with Western Time Zones).
+  - Join us from 15:00 - 16:00 ([see in your local time](http://time.unitarium.com/utc/14)) London time on 14 and 28 July 2021 (compatible with Eastern Time Zones).
+
+## News from the Community
+
+A month after the Book Dash (May 2021), several of our participants have merged the first versions of their chapters on _The Turing Way_ guides.
```

Comment:
at the end of the sentence - should it say 'into' rather than on the turing way guides?